.github/workflows/release-mockotlpserver.yml

```diff
@@ -29,6 +31,43 @@ jobs:
           node-version: 'v18.20.4'
           registry-url: 'https://registry.npmjs.org'
 
+      # Push a Docker image.
+      - uses: docker/setup-buildx-action@v3
+      - id: docker-meta
+        uses: docker/metadata-action@369eb591f429131d6889c46b94e711f089e6ca96  # v5.6.1
+        with:
+          images: ${{ env.DOCKER_IMAGE_NAME }}
+          flavor: |
+            latest=auto
+          tags: |
+            # "1.2.3" and "latest" Docker tags on push of git tag "v1.2.3"
+            type=semver,pattern={{version}}
+          labels: |
+            org.opencontainers.image.vendor=Elastic
+            org.opencontainers.image.title=mockotlpserver
+            org.opencontainers.image.description=A mock OTLP server, for dev and testing
+      - uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
```

Comment:
this is a bit different from the release of the `@elastic/opentelemetry-node`. Just curious bout it

Reply:
Yah, ghcr is GitHub's own container registry. It is easy, for auth at least, to publish to it. We are using it for publishing a light fork of Ollama for some of the GenAI work.  Publishing to docker.elastic.co requires more hoops and might imply more of a supported image than we care to have just for mockotlpserver as a dev tool.